include/rdm/param_data.h

```diff
@@ -0,0 +1,585 @@
+  uint32_t list_change_number;
+  RdmUid uids[RDMPD_MAX_ENDPOINT_RESPONDER_LIST_ENTRIES];
+  size_t number_of_uids;
+} RdmPdEndptResponderList;
+
+/*! The maximum number of endpoint responder list UIDs in an ACK overflow response. */
+#define RDMPD_MAX_ENDPOINT_RESPONDER_LIST_OVERFLOW_ENTRIES 38
+
+typedef struct RdmPdEndptResponderListOverflow
+{
+  RdmUid uids[RDMPD_MAX_ENDPOINT_RESPONDER_LIST_OVERFLOW_ENTRIES];
+  size_t number_of_uids;
+} RdmPdEndptResponderListOverflow;
+
+etcpal_error_t rdmpd_unpack_get_endpt_resp_list(const RdmParamData *pd, uint16_t *endpt_id_out);
+etcpal_error_t rdmpd_pack_get_endpt_resp_list(uint16_t endpt_id, RdmParamData *pd_out);
+etcpal_error_t rdmpd_unpack_get_resp_endpt_resp_list(const RdmParamData *pd, RdmPdEndptResponderList *list_out);
+etcpal_error_t rdmpd_pack_get_resp_endpt_resp_list(const RdmPdEndptResponderList *list, RdmParamData *pd_out);
+etcpal_error_t rdmpd_unpack_get_resp_endpt_resp_list_overflow(const RdmParamData *pd,
+                                                              RdmPdEndptResponderListOverflow *list_out);
+etcpal_error_t rdmpd_pack_get_resp_endpt_resp_list_overflow(const RdmPdEndptResponderListOverflow *list,
+                                                            RdmParamData *pd_out);
+
+// Get ENDPOINT_RESPONDER_LIST_CHANGE
+typedef struct RdmPdEndptRespListChange
+{
+  uint16_t endpoint_id;
+  uint32_t list_change_number;
+} RdmPdEndptRespListChange;
+
+etcpal_error_t rdmpd_unpack_get_endpt_resp_list_change(const RdmParamData *pd, uint16_t *endpoint_id_out);
+etcpal_error_t rdmpd_pack_get_endpt_resp_list_change(uint16_t endpoint_id, RdmParamData *pd_out);
+etcpal_error_t rdmpd_unpack_get_resp_endpt_resp_list_change(const RdmParamData *pd,
+                                                            RdmPdEndptRespListChange *change_out);
+etcpal_error_t rdmpd_pack_get_resp_endpt_resp_list_change(const RdmPdEndptRespListChange *change, RdmParamData *pd_out);
+
+// Get BINDING_CONTROL_FIELDS
+typedef struct RdmPdEndpointUid
+{
+  uint16_t endpoint;
+  RdmUid uid;
+} RdmPdEndpointUid;
+
+typedef struct RdmPdBindingControlFields
+{
+  RdmPdEndpointUid endpoint_uid;
+  uint16_t control_field;
+  RdmUid binding_uid;
+} RdmPdBindingControlFields;
+
+etcpal_error_t rdmpd_unpack_get_binding_control_fields(const RdmParamData *pd, RdmPdEndpointUid *endpoint_uid_out);
+etcpal_error_t rdmpd_pack_get_binding_control_fields(const RdmPdEndpointUid *endpoint_uid, RdmParamData *pd_out);
+etcpal_error_t rdmpd_unpack_get_resp_binding_control_fields(const RdmParamData *pd,
+                                                            RdmPdBindingControlFields *binding_ctrl_fields_out);
+etcpal_error_t rdmpd_pack_get_resp_binding_control_fields(const RdmPdBindingControlFields *binding_ctrl_fields,
+                                                          RdmParamData *pd_out);
+
+// Get/Set BACKGROUND_QUEUED_STATUS_POLICY
+typedef enum
+{
+  kRdmPdBqspsStatusNone = 0x00,
```

Comment:
Haha, wow, the abbreviation here is a little rough. @nwagner Any thoughts on how to make this better?